src/html/options.html

```diff
@@ -23,6 +23,16 @@ <h4 class="label"><a href="https://betterlibrarything.com/docs/librarian/tag-ind
 	<button id="save-tag-index" class="log-button colour-default" disabled>Save</button>
 </div>
 
+<div class="form">
+	<h4 class="label">Enfore logging in and setting tag index before using LibraryThing?</h4>
```

Comment:
Can we use name it `Tag Index`? Just to be consistent with the rest of our docs

Reply:
Actually this could say `Enforce Tag Index Access` too.

Reply:
(also typo: "Enfore")

Reply:
LAST REQUEST: can we spawn in a big ugly warning in this form when you disable this feature?

Like... just an element that defaults to display:none, and is shown iff toggle is set to false?